tile/targets/cpu/jit.cc

```diff
@@ -202,10 +204,11 @@ class Compiler : private stripe::ConstStmtVisitor {
   std::map<std::string, Scalar> scalars_;
   std::map<std::string, Buffer> buffers_;
   std::map<std::string, Index> indexes_;
+  uint64_t arenaSize_;
```

Comment:
given that there are two different constructor methods, perhaps it'd be safer to use value-initialization here?

Reply:
Good catch! Thanks!